src/lib/DeviceCore.Abstractions/DeviceCore.Abstractions.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<Nullable>enable</Nullable>
+		<RootNamespace>DeviceCore</RootNamespace>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="9.0.6" />
+		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
+		<PackageReference Include="System.Reactive" Version="6.0.1" />
```

Comment:
I don't know if it's a good idea to pull System.Reactive as a dependency in the Abstractions package.

Reply:
You are right, it should probably be in the implementation only, since it doesn't mean every implementation will need it.

Reply:
It's something we've avoided in other packages that we've done.